taskwise/src/pages/Board/Board.jsx

```diff
@@ -1,78 +1,179 @@
 import React from 'react';
-import { Box, Container, Paper } from '@mui/material';
-import Column from "./Column"
+import { useParams,useNavigate } from 'react-router-dom';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import Column from "./Column";
+import { useState } from 'react';
+import { Box, Paper, Typography, Grid } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { styled } from '@mui/material/styles';
+import InputBase from '@mui/material/InputBase';
+import Divider from '@mui/material/Divider';
+import { Button } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+
+const SearchIconWrapper = styled('div')(({ theme }) => ({
+    padding: theme.spacing(0, 2),
+    height: '100%',
+    position: 'absolute',
+    pointerEvents: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+}));
+
+const StyledInputBase = styled(InputBase)(({ theme }) => ({
+    color: 'inherit',
+    width: '100%',
+    '& .MuiInputBase-input': {
+        padding: theme.spacing(1, 1, 1, 0),
+        paddingLeft: `calc(1em + ${theme.spacing(4)})`,
+        transition: theme.transitions.create('width'),
+        width: '100%', // Start with full width within its container
+    },
+}));
+
+const Search = styled('div')(({ theme }) => ({
+    position: 'relative',
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: "#f0f0f0",
+    marginLeft: 0,
+    paddingRight: theme.spacing(2), // Add right-side padding
+    width: '200px', // Default width
+    height: '30px', // Fixed height
+    display: 'flex',
+    alignItems: 'center',
+    transition: theme.transitions.create('width'), // Animate width change
+    '&:focus-within': {
+        width: '300px', // Expanded width on focus
+    },
+}));
+
+
+
 function Board() {
-    //const { id } = useParams();
+    const { id } = useParams();
+    const navigate=useNavigate();
+    const handleClick=()=>{
+        navigate(`/projects/${id}/new-task`)
+    }
+    const project = {
+        img: 'https://img.freepik.com/free-vector/hand-drawn-minimal-background_23-2149001650.jpg?t=st=1716280160~exp=1716280760~hmac=f254cfeda21a263638253b9f6f0c0c9028bac218840dea34d6de5739054a4a96'
+    };
     const initialData = {
-        "order": [1, 2, 3],
-        "columns": {
-            "1": {
-                "id": 1,
-                "title": "To Do",
-                "taskIds": [1, 2]
-            },
-            "2": {
-                "id": 2,
-                "title": "In Progress",
-                "taskIds": [3]
-            },
-            "3": {
-                "id": 3,
-                "title": "Done",
-                "taskIds": [4]
-            }
+        order: [1, 2, 3, 4, 5, 6],
```

Comment:
Hi @navina1 , Please reply to this thread. it will have history.

https://github.com/AbhishekGogoi/Taskwise_frontend/pull/6#issuecomment-2132123015

for handling dummy data in a React app, Even if we are using it before integrating with the backend
it is recommended to Use Separate JSON Files, to follow best practices for maintaining clean, organized, and scalable code